src/server/webserver.py

```
@@ -129,6 +138,17 @@ async def refine_code(request: RefinementRequest):
     except Exception as e:
         logger.error(f"Refinement error: {e}")
         raise HTTPException(status_code=500, detail=str(e))
+    
+@app.post("/autocomplete")
+async def autocomplete_code(request: CompletionRequest):
+    try:
+        completed_code = autocomplete_instance.predict_completion(request.code, request.context)
+        logger.info(f"Autocomplete prediction:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)